### ISSUE TITLE
error when compiling from source 

### ISSUE THREAD

**Issue by deartonym** [openslide/openslide#208]
I got the following error when I try to compile it from source under CentOS 6.3:

`src/openslide-dll.rc:1:21: fatal error: winuser.h: No such file or directory`

**Issue type** (fatal error during make): 
**Operating system** (CentOS 6.3): 
**Platform** (64-bit x86): 
**OpenSlide version**: 3.4.1 

It seems that it want to compile 'src/openslide-dll.lo' which should not be compiled in Linux systems.

Anyone knows how to solve this?

Best,

**Comment by bgilbert** [comment]
OpenSlide should only try to build `openslide-dll.rc` when `configure` is run with `--host=<something>-mingw32`.  Could you post your `config.log`?

**Comment by bgilbert** [comment]
Closing due to lack of response.  If you're still seeing this, please feel free to reopen.

**Comment by agam-kashyap** [comment]
Hey, I am getting the same error. I am attaching my config file
System: PopOs 20.04

[config.log](https://github.com/openslide/openslide/files/6841786/config.log)

**Comment by bgilbert** [comment]
Thanks for the log.  You're hitting a bug which was fixed by a3153fa9fb8e37e14941f441a1647532a6ceaa9a after 3.4.1 was released.  You can avoid it by removing `windres` from your PATH or by building OpenSlide from Git.

Note that building from Git has a few additional dependencies.  From the README:

```
(If building from the Git repository, you will first need to install
autoconf, automake, libtool, and pkg-config and run "autoreconf -i".)
```
